jerry-core/ecma/base/ecma-gc.c

```diff
@@ -180,16 +137,20 @@ ecma_gc_set_object_visited (ecma_object_t *object_p, /**< object */
     is_visited = !is_visited;
   }
 
-  object_p->container = JRT_SET_BIT_FIELD_VALUE (uint64_t, object_p->container,
-                                                 is_visited,
-                                                 ECMA_OBJECT_GC_VISITED_POS,
-                                                 ECMA_OBJECT_GC_VISITED_WIDTH);
+  if (is_visited)
```

Comment:
Aren't the above 5 lines simply equal to `if (is_visited != ecma_gc_visited_flip_flag)`?